### ISSUE TITLE
Problem in getting comments

### ISSUE THREAD

**Issue by sedehi** [postaddictme/instagram-php-scraper#43]
I am using the below code for receiving comments and the post shows 684 comments right now but it returns me 861 comments, obviously different quantity.
Also when I make comments unique, there would be 290 unique comments... Do you have any idea where the problem is?

`
 $comments = InstagramScraper\Instagram::getMediaCommentsByCode('BMJbCHuDAAp', 1000);
`

**Comment by raiym** [comment]
Hello. Sorry for long reply. 
Just now I have checked by hands this post: https://www.instagram.com/p/BP2xB2qF3vJ/
getMediaCommentsByCode response contatins 166 comments right now, but instagram website says that number of comments is 172.
I have checked that every comment shown on webpage exists in `$comments = InstagramScraper\Instagram::getMediaCommentsByCode('BP2xB2qF3vJ', 200); ` and vice versa.
I think it is a bug on Instagram website. 

About uniqueness. How you check if it is unique? By id?
